microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -121,9 +121,22 @@ The following functions related to checking workload health in the `/usr/share/m
+
+* Previously, specific cipher suites required by {microshift-short} etcd when using TLS 1.2 were not included in the {microshift-short} startup configuration file. As a result, {microshift-short} failed to start when using TLS 1.2. With this release, the required cipher suites are in the configuration file by default and {microshift-short} can start normally when using TLS 1.2. (link:https://issues.redhat.com/browse/OCPBUGS-48735[OCPBUGS-48735])
+
+* Previously, a `.nodename` file that holds the last hostname was created by MicroShift on startup non-atomically. When MicroShift's startup was interrupted, the `.nodename` file was left behind empty. This `.nodename` file was used on the next MicroShift startup, causing the node name to stored as an empty string. This caused the API Server to reject the kubelet's calls and the start up failed. With this release, the `.nodename` file is created atomically with each MicroShift start up, preventing the error. (link:https://issues.redhat.com/browse/OCPBUGS-48163[OCPBUGS-48163])
+
+* By default, the {microshift-short} Logical Volume Manager Storage (LVMS) image copied the manifest list for four platforms because of automatic digest preservation. This action caused unnecessary usage of disk and network space. With this release, the manifest list is replaced with images specific to each architecture. Now, the {microshift-short} LVMS images only contain supported platform images, saving on disk space and network bandwidth. (link:https://issues.redhat.com/browse/OCPBUGS-51329[OCPBUGS-51329])
+
+* Previously, the `kustomizer` sub-service was blocking `microshift.service` readiness by adding a manifest that required a webhook for a Custom Resource (CR) before {microshift-short} and associated network services had started. This resulted in both `kustomizer` and the network node failing, causing {microshift-short} to fail. With this release, {microshift-short} is no longer dependent on the `kustomizer` sub-service, and can start network services and be ready before the application service starts. (link:https://issues.redhat.com/browse/OCPBUGS-51365[OCPBUGS-51365])
+
+* In previous versions of {microshift-short} and {op-system-base}, images built with image builder in {op-system-base} 9.4 and {microshift-short} embedded containers could bypass correct updates because of a difference in the directories in which those images were stored. This could prevent updates for disconnected and offline deployments. With this release, and {op-system-base} 9.6, the issue is resolved. Updates for {op-system-image} are applied as expected with {microshift-short} and embedded containers. (link:https://issues.redhat.com/browse/OCPBUGS-52420[OCPBUGS-52420])
```

Comment:
We should rephrase `could bypass correct updates because of a difference in the directories in which those images were stored`. 

```suggestion
* In previous versions of {microshift-short} and {op-system-base}, {microshift-short} container image embedding procedure was advertised to use the default container storage directory, which prevented proper image updates. With this release, the recommended image embedding procedure was changed to use a dedicated directory per each image and a systemd service copying those embedded images into the default container storage.
(link:https://issues.redhat.com/browse/OCPBUGS-52420[OCPBUGS-52420])
```